src/snowflake/snowpark/session.py

```diff
@@ -279,25 +279,6 @@ def _get_sandbox_conditional_active_session(session: "Session") -> "Session":
     return session
 
 
-def _close_session_atexit():
```

Comment:
I'm curious why we need to reorder -- what issue has this ordering caused?

Reply:
see https://github.com/snowflakedb/snowpark-python/pull/2566#discussion_r1830096835